Readme.md

```diff
@@ -11,6 +11,9 @@ built from here:
 
 ------------------------
 
+# Usage
+A fully working sample project can be found at https://github.com/treblereel/j2cl-tests
```

Comment:
separate PR for unrelated changes please - though I have dmitrii's old archetype getting ready to merge, we'll want to call out both.

Reply:
This one confuses me, wasn't this one-liner already accepted as a separate PR before?

Reply:
It is possible you are not merged up to date?

Be sure you are using "main" and not "master", i'm using github's "change the main branch" workflow to change that. If you are using your IDE's git tooling, this might not be visible, I'm not sure.

Reply:
It was this one https://github.com/Vertispan/j2clmavenplugin/pull/85